.github/workflows/create-release.yml

```diff
@@ -408,9 +408,9 @@ jobs:
     runs-on: ubuntu-22.04
     steps:
     - name: Setup Go
-      uses: actions/setup-go@v3
+      uses: actions/setup-go@v5
       with:
-        go-version: 1.20.x
+        go-version: stable
```

Comment:
Is this what we do in other scripts ?

Reply:
Yes, this one is on github-config too and the reason i believe is set to stable and not to a specific minor version, is because the paketo-bot upgrades the go version in the acceptance tests. So seems necessary to always be on stable version, otherwise the acceptance tests might break as they get updated from paketo-bot e.g. in our case this is what caused the failure on this step https://github.com/paketo-buildpacks/jammy-tiny-stack/commit/76ca12a204f25dce24627ce5fa0e3a85fc8e1973